build/edge/run_daemon.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
We need some documentation in the readme as to how to run edge_core in a container.  Can we have the documentation as a part of this PR ?

Reply:
The documentation is along with the #248 . If it makes more sense to commit docs along with this PR, I'm glad to do a correction.

Reply:
oh sorry I missed it. Its ok to have it as part of #248 . thanks !